pimcore/models/Element/Service.php

```diff
@@ -759,7 +759,11 @@ public static function createFolderByPath($path, $options = [])
 
         if (!($foundElement = $type::getByPath($sanitizedPath))) {
             foreach ($parts as $part) {
-                $pathsArray[] = $pathsArray[count($pathsArray) - 1] . '/' . self::getValidKey($part, $type);
+                $pathPart = '';
```

Comment:
@mbolka IMHO this can be a little better, don't you think?:
```php
$pathPart = $pathsArray[count($pathsArray) - 1] ?? '';
$pathsArray[] = $pathPart . '/' . self::getValidKey($part, $type);
```